### PR TITLE
Db 723 validator logging

### PR DESCRIPTION
**Pull request by klrBAH**

### CODE REVIEW

File: dataactvalidator/validation_handlers/validationManager.py [hunk 1]
```diff
@@ -221,8 +223,9 @@ def runValidation(self, jobId, interfaces):
                                 self.reportHeaders) as writer:
                 while not reader.isFinished:
                     rowNumber += 1
-                    #if (rowNumber % 1000) == 0:
-                    #    print("Validating row " + str(rowNumber))
+                    if (rowNumber % 100) == 0:
+                        CloudLogger.logError("VALIDATOR_INFO: ","JobId: "+str(jobId)+" validating/copying row " + str(rowNumber),"")
```
Comment (jworcestBAH) [inline dataactvalidator/validation_handlers/validationManager.py]: We're moving validation out of this part, a better message might be "loading row"

**Comment by jworcestBAH** [comment]
Added a few questions about messages, also need to figure out why Jenkins tests are failing.

**Comment by klrBAH** [comment]
Updated messages as requested and fixed TypeError causing unit tests to fail.

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
@jworcestBAH unit tests completed successfully. Can we merge?

**Comment by jworcestBAH** [comment]
Changes look good, merging now

**Pull request merged**